### PR TITLE
Retrospective consistency for welcome.md

### PR DESCRIPTION
**Pull request by paulowoicho**
### Summary

Fixes #1174 

In this PR, I edited the welcome page of the book to ensure consistent grammar and tone. I also converted the image from markdown to MyST

### List of changes proposed in this PR (pull-request)

* Edited to ensure good, consistent grammar
* Updated image from markdown to MyST format

### What should a reviewer concentrate their feedback on?

- [ ] Do the edits help make the chapter easier to read/understand?
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/welcome.md [hunk 1]
```diff
@@ -1,22 +1,28 @@
+(welcome)=
 # Welcome
 
-_The Turing Way_ is an open source community-driven guide to reproducible, ethical, inclusive and collaborative data science.
+_The Turing Way_ is an open-source community-driven guide to reproducible, ethical, inclusive and collaborative data science.
```
Comment (malvikasharan) [inline book/website/welcome.md:4]:
```suggestion
_The Turing Way_ is an open source community-driven guide to reproducible, ethical, inclusive and collaborative data science.
```

Comment (malvikasharan) [reply book/website/welcome.md:4]: We use open source without hyphenation, though I know Grammarly doesn't understand! :D I'd suggest ignoring Grammarly suggestion on this one.

**Review by malvikasharan — CHANGES_REQUESTED** [review]
Looks great!

**Review by malvikasharan — APPROVED** [review]
With these suggestions integrated, I am happy to let you merge this. Great job!

File: book/website/welcome.md (hunk 1 above)
Comment (paulowoicho) [reply book/website/welcome.md]: Thanks! I will keep this in mind!

**Pull request merged**